spec/features/idv/steps/in_person/state_id_50_50_spec.rb

```diff
@@ -0,0 +1,378 @@
+require 'rails_helper'
+
+RSpec.describe 'state id 50/50 state', js: true, allowed_extra_analytics: [:*],
```

Comment:
Depending if #11099 is merged before this one, we should rebase and check whether `allowed_extra_analytics` is still necessary. Ideally we'd not be adding new instances of this, as it's meant for legacy only.

Reply:
Okay thanks for the heads up! I can rebase this branch off of that branch using `git rebase --onto` and try it out.

Reply:
After rebasing with your branch, it appears that the `allow_extra_analytics` is still needed. When I removed `allowed_extra_analytics` from the describe block I receive the following error message:

```
FakeAnalytics::UndocumentedParams:
              event :idv_doc_auth_welcome_visited called with undocumented params [:opted_in_to_in_person_proofing]
              (if these params are for specs only, use :allowed_extra_analytics metadata)
```

Reply:
The `opted_in_to_in_person_proofing` property seems to be coming from the `ab_test_analytics_buckets` method if that helps.

Reply:
Discussed on Slack, but we can either add the missing analytics event property documentation, or re-add `allowed_extra_analytics`. Ideally we're documenting all analytics methods, but I think it'd be fair to call it out of scope.